### DISCUSSION TITLE
How speed up app in Google PageSpeed Insights?

### DISCUSSION THREAD

**Discussion by Kogomre** [vercel/next.js#17941] [Help]
Hi, we have a serious performance issue when we test the loading speed in Google PageSpeed Insights. We have read a lot of guides, but we still haven't found a solution how to make the download speed index on mobile devices increase. As a test, we used one page, where everything was connected via dynamic import, but this did not speed up. My development team is at an impasse, and we don't know where to go next. Maybe there is some kind of solution that is not written anywhere?
Screenshot with pagespeed rating
![screenshot](https://user-images.githubusercontent.com/43267690/96253799-eb3f6200-0fbc-11eb-813a-f23c1843d3ae.jpg)

**Comment by nikola1970** [comment]
Also looking forward to this problem. Have even lower than that.

**Comment by ljosberinn** [comment]
You seem to simply have way too many dependencies. Reconsider what you're including and what you don't need. This has nothing to do with Next.js. 600kB gzipped bundles are way too much. Also Lighthouse tells you you should try to reduce your DOM size.

**Reply by njho** [reply]
Lol as ridiculous as it seems, this was the answer I needed to start looking at bundle size
